.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: node_js
 
 node_js:
-  - "4"
-  - "6"
-  - "7"
+  - "8"
```

Comment:
Since the whole point of this silly PR is node8, it made sense to drop support for older versions.